README.md

```diff
@@ -636,4 +636,87 @@ AlternateEndUserIdentifierResponse response = Aeui.get(client, id);
 ### List Aeuis
 ```java 
 AlternateEndUserIdentifiersResponse response = Aeui.list(client, query);
+```
+
+## Tn Options
+
+### Tn Options Create
+
+Creates a TN Option order to assign line features to the telephone number.
+
+#### Create PortOutPasscode
+
+```java
+TnOptionOrder order = new TnOptionOrder();
+order.setTnOptionGroups(new ArrayList<>());
+order.setCustomerOrderId("TnOptionOrder1");
+
+TnOptionGroup optionGroup = new TnOptionGroup();
+optionGroup.setPortOutPasscode("12abd38");// PortOutPasscode set here
+optionGroup.setTelephoneNumbers(new ArrayList<>());
+optionGroup.getTelephoneNumbers().add("2018551020");
+
+order.getTnOptionGroups().add(optionGroup);
+
+TnOptionOrderResponse response = TnOptions.create(client, order);
+```
+
+#### Create Call Forward Number
+
+```java
+TnOptionOrder order = new TnOptionOrder();
+order.setTnOptionGroups(new ArrayList<>());
+order.setCustomerOrderId("TnOptionOrder1");
+
+TnOptionGroup optionGroup = new TnOptionGroup();
+optionGroup.setCallForward("2018551022");// Call Forward set here
+optionGroup.setTelephoneNumbers(new ArrayList<>());
+optionGroup.getTelephoneNumbers().add("2018551020");
+
+order.getTnOptionGroups().add(optionGroup);
+
+TnOptionOrderResponse response = TnOptions.create(client, order);
+```
+
+#### Enable SMS
+
+```java
+TnOptionOrder order = new TnOptionOrder();
+order.setTnOptionGroups(new ArrayList<>());
+order.setCustomerOrderId("TnOptionOrder1");
+
+TnOptionGroup optionGroup = new TnOptionGroup();
+optionGroup.setSms("true");// SMS set true here
```

Comment:
I don't think this is a valid option. `Sms - [ on | off | unchanged]` is what the docs say

Reply:
Ooops you are right